### ISSUE TITLE
Getting started with flux should provide a working example

### ISSUE THREAD

**Issue by rbetts** [influxdata/docs.influxdata.com#2912]
###### URL for relevant page?

https://docs.influxdata.com/flux/v0.65/guides/execute-queries/

###### What products and version are you using?

InfluxDB 1.8.1 OSS

###### Where did you look before opening the issue?

This was the first google hit.  The docs show how to start the repl but don't show a valid query.  It would be great to show a working example so that a novice user can validate that it is functional... and get started.

**Comment by timhallinflux** [comment]
yeah...there is a simple example in the API section just below.  Perhaps we simply re-use that one for the REPL example?

```from(bucket:"telegraf")
        |> range(start:-5m)
        |> filter(fn:(r) => r._measurement == "cpu")```
